sdk/spring/CHANGELOG.md

```diff
@@ -31,44 +21,51 @@ This section includes changes in `spring-cloud-azure-autoconfigure` module.
 - Delete the auto configuration for Spring Cloud Sleuth support[#31543](https://github.com/Azure/azure-sdk-for-java/pull/31543).
 - Use the `com.nimbusds.jwt.proc.DefaultJWTClaimsVerifier#DefaultJWTClaimsVerifier(com.nimbusds.jwt.JWTClaimsSet, java.util.Set<java.lang.String>)` instead of `com.nimbusds.jwt.proc.DefaultJWTClaimsVerifier#DefaultJWTClaimsVerifier()`[#31543](https://github.com/Azure/azure-sdk-for-java/pull/31543).
 - Use the `org.springframework.http.ResponseEntity#getStatusCode` instead of `org.springframework.http.ResponseEntity#getStatusCodeValue`[#31543](https://github.com/Azure/azure-sdk-for-java/pull/31543).
-- Use the `com.azure.core.util.ConfigurationBuilder#ConfigurationBuilder()` instead the default constructor method `com.azure.core.util.Configuration#Configuration()`[#31543](https://github.com/Azure/azure-sdk-for-java/pull/31543).
```

Comment:
Why delete this one? It's better to add description in the PR description.

Reply:
This change can be ignored by developers, it's used for `sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/factory/AbstractAzureServiceClientBuilderFactory.java`.